builder/vsphere/supervisor/config.go

```diff
@@ -0,0 +1,62 @@
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type Config
+
+package supervisor
+
+import (
+	packercommon "github.com/hashicorp/packer-plugin-sdk/common"
+	"github.com/hashicorp/packer-plugin-sdk/communicator"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-sdk/template/config"
+	"github.com/hashicorp/packer-plugin-sdk/template/interpolate"
+
+	"github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/common"
+)
+
+const (
+	defaultSSHUsername = "root"
```

Comment:
I spoke with Mike and William offline and both confirmed that there was no standard default SSH username provided (i.e. it's normally user defined). I am going to keep this as it's required for the Packer builder but will change it to a different name rather than "root".

Reply:
Changed it to `default-user` in the latest patch.